cypress/e2e/homepage.cy.js

```diff
@@ -15,10 +15,42 @@ describe('Test cases for W&Co', () => {
     cy.url().should('eq', 'https://work.co/clients/');
   })
   
-  it('Select Expertise and Capabilities',()=>{
+  it('Select Practise Areas',()=>{
     cy.get('[data-test-id="global-menu-btn"]').click()
     cy.get('[data-test-id="grid-practice-areas-link"]').click()
     cy.url().should('eq', 'https://work.co/company/');
   })
+  it('Go back to homepage',()=>{
+    cy.get('[data-test-id="global-menu-btn"]').click() 
+  })
+  it('Subscribe to Newsletter',()=>{
+    cy.get('[data-test-id="footer-subscribe-newsletter-link"]').click()
+    cy.get('[data-test-id="newsletter-subscribe-submit"]').should('be.disabled')
+    cy.get('[data-test-id="newsletter-email-textfield"]').click().type('zivanovic@work.co')
+    cy.get('[data-test-id="newsletter-subscribe-submit"]').should('be.enabled')
+
+
+  })
+  it('Check Disclaimer',()=>{
+    cy.get('.Subscribe-gdprDisclaimer').should('contain','We only send emails when we have something to say.')
+    cy.get('a').contains('Privacy Policy').click()
```

Comment:
instead of .click(), take the href attributes of these 2 buttons and assert that they equal to links provided